project/plugins.sbt

```diff
@@ -1,6 +1,6 @@
 resolvers += "Typesafe repository" at "https://repo.typesafe.com/typesafe/releases/"
 
-// sbt-native-packager cannot be updated to >1.10.3 until Play supports scala-xml 2
+// sbt-native-packager cannot be updated to >1.10.4 until Play supports scala-xml 2
```

Comment:
This is probably the change that wants investigating, here! Let's drill in before this gets merged, ideally running it locally to check it starts properly.

Reply:
This is weird. The version of `sbt-native-packager` was updated 3 months ago (seemingly by Scala Steward) here: https://github.com/guardian/janus-app/commit/bf3514c1fb6eaf5f74df119b44219cde96369341, but there is a comment to ignore this package (`// scala-steward:off`)so not sure why this happened? Is the syntax of the ignore comment wrong? Secondly this earlier change updates the comment version from 1.0.1 to 1.0.3: https://github.com/guardian/janus-app/commit/21895023cd539c5c8846fab6a28ba59caa59d90d - I guess we should either have the version in the comment also ignored, or format it such that it's not recognised as a version by Scala Steward?

Reply:
Right yeah, so this is a non-issue then. We can remove this comment, maybe worth us doing that here while we're thinking about it?